packages/react/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  coverageReporters: ['text-summary', 'lcov'],
```

Comment:
`collectCoverageFrom` config is required I believe. If I remember correctly, if it is not provided, if a target file is not referenced by any test file, it won't be counted into the coverage collection.

Reply:
Will use `coveragePathIgnorePatterns: ['/lib/', '/node_modules/']` instead.

Reply:
Updated. PTAL.